### ISSUE TITLE
Sudden close of Telegraf caused by OPC UA input plugin

### ISSUE THREAD

**Issue by hybuild-project** [influxdata/telegraf#10140]
### Relevent telegraf.conf

```toml
[[inputs.opcua]]
  name = "xyz"
  endpoint = "opc.tcp://xxx.yyy.wwww.zzz:4840"
  security_policy = "None"
  security_mode = "None"
  nodes = [
    { name="abc",  namespace="6",  identifier_type="s",  identifier="abc"},
    { name="def",  namespace="6",  identifier_type="s",  identifier="def"},
...
]
```

### System info

Telegraf 1.20.3, Windows 10

### Steps to reproduce

1. Simly running Telegraf with that plugin
2.
3.
...

### Expected behavior

No errors

### Actual behavior

After some random time (hours, one day at most) it crashes:

goroutine 68418 [running]:
github.com/gopcua/opcua/uasc.(*SecureChannel).dispatcher(0xc0007b8000)
        /go/pkg/mod/github.com/gopcua/opcua@v0.2.0-rc2.0.20210409063412-baabb9b14fd2/uasc/secure_channel.go:208 +0x185
created by github.com/gopcua/opcua/uasc.(*SecureChannel).open.func1
        /go/pkg/mod/github.com/gopcua/opcua@v0.2.0-rc2.0.20210409063412-baabb9b14fd2/uasc/secure_channel.go:496 +0x5f

**Comment by R290** [comment]
I'm facing a similar issue after upgrading Telegraf to version 1.20.4:

```
2021-11-23T13:05:10Z E! [inputs.opcua] Error in plugin: RegisterNodes Read failed: The operation could not complete because the client is not connected to the server. StatusBadServerNotConnected (0x800D0000)
panic: send on closed channel

goroutine 8337 [running]:
github.com/gopcua/opcua/uasc.(*SecureChannel).dispatcher(0xc000cbec40)
        /go/pkg/mod/github.com/gopcua/opcua@v0.2.0-rc2.0.20210409063412-baabb9b14fd2/uasc/secure_channel.go:208 +0x185
created by github.com/gopcua/opcua/uasc.(*SecureChannel).open.func1
        /go/pkg/mod/github.com/gopcua/opcua@v0.2.0-rc2.0.20210409063412-baabb9b14fd2/uasc/secure_channel.go:496 +0x5f
```

There is an extra line at the beginning about sending on a closed channel. In my case this is caused by the PLC closing the TCP connection. Let me see if I can find out what changed between versions. Could be Telegraf itself or the [gopcua](https://github.com/gopcua/opcua) library.

**Comment by hybuild-project** [comment]
Very nice investigation R290! Let-s hope it will be included in v1.21...! Cheers and thank you so much.
